server/protocols/webadmin/webadmin-cassandra-data/src/main/java/org/apache/james/webadmin/service/CassandraMappingsSolveInconsistenciesTask.java

```diff
@@ -34,14 +34,18 @@
     public static final String TYPE = "cassandraMappingsSolveInconsistencies";
 
     public static class AdditionalInformation implements TaskExecutionDetails.AdditionalInformation {
-        private final long processedMappingsCount;
+        private final MappingsSourcesMigration.AdditionalInformation info;
 
-        AdditionalInformation(long processedMappingsCount) {
-            this.processedMappingsCount = processedMappingsCount;
+        AdditionalInformation(MappingsSourcesMigration.AdditionalInformation info) {
+            this.info = info;
         }
 
-        public long getProcessedMappingsCount() {
-            return processedMappingsCount;
+        public long getSuccessfulMappingsCount() {
```

Comment:
where We will use 2 `get` method?

Reply:
When getting the additional information of the task. It's a hack on top of Jackson, as Benoit explained to me